/**:
   ros__parameters:
      platform_height: 0.05
      wheel_radius: 0.3
      max_velocity: 0.22
      acceleration: 9.8